webanno-io-tsv/src/main/java/de/tudarmstadt/ukp/clarin/webanno/tsv/util/AnnotationUnit.java

```diff
@@ -49,20 +49,18 @@ public int hashCode() {
 	}
 
 	@Override
-	public boolean equals(Object obj) {
-		if (this == obj)
+	public boolean equals(Object obj)
+	{
+		if (this == obj) {
 			return true;
-		if (obj == null)
+		}
```

Comment:
They seem to be ok. IntelliJ probably auto-corrected their placement because they didn't fit to the code style.